services/primary-composite-key.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var _ = require('lodash');
+
+function PrimaryCompositeKeys(model, schema, record) {
```

Comment:
This is a `CompositeKeysManager` with which you can:

- generate the "fake" id of a record (the method `get` you defined but that is not really well-name IMO)
- extract the conditions to retrieve the record (the method you should have factorized here and that is duplicated in `resource-getter` and `resource-updater`)